src/packages/analytics/index.tsx

```diff
@@ -6,13 +6,13 @@ import { version as APP_VERSION } from 'constants/version';
 import { useAuth } from 'context/AuthContext';
 
 const UserIdentifier: React.FC = () => {
-  const { accountAddress } = useAuth();
+  const { accountAddress, chainId } = useAuth();
   const posthog = usePostHog();
 
   // Identify user by their account address
   useEffect(() => {
```

Comment:
You'll need to pass `chainId` to the list of dependencies so that the effect triggers when it changes

Reply:
Oops, fixed it